packages/bsky/tests/notification-server.test.ts

```diff
@@ -1,231 +0,0 @@
-import AtpAgent, { AtUri } from '@atproto/api'
-import { TestNetwork, SeedClient } from '@atproto/dev-env'
-import basicSeed from './seeds/basic'
-import { NotificationServer } from '../src/notifications'
-import { Database } from '../src'
-
-describe('notification server', () => {
```

Comment:
any idea the future of this stuff? it'll be handled by the dataplane indexer i guess? should just make a note so we don't forget about it 😅

Reply:
Indeedy, looking like it will move into the indexer.